app/views/qsets/_question_page_header_large_screen.html.erb

```diff
@@ -9,9 +9,14 @@
     <%= render "all_mine_other_filter" %>
   </div>
 </div>
-<div class="row col-xs-12">
+<div class="row">
   <%= render "breadcrumbs" %>
 </div>
-<div class=" row col-xs-12 center">
-  <a href="#" class="btn btn-primary btn-large quiz-all" id="quizAllButtonID"> Quiz All </a>
+<div class="row center">
```

Comment:
(similar to [the one on the small screen partial](https://github.com/penzance/askup/pull/85/commits/ef62469e1ffaa10ab399a9b5c2c20d8af6d633d6?diff=split#diff-3db5bd34faef53f18da0c4a43218fa8aR22), but you wouldn't need the extra classes of course)